### PR TITLE
Support `const`, `static` and `type`

### PR DESCRIPTION
**Pull request by loyd**

### CODE REVIEW

**Comment by loyd** [comment]
@sagebind, look at PR when you can, please, it's needed for releasing a new version of `elfo` crate.

File: src/unstable.rs [hunk 1]
```diff
@@ -21,13 +21,16 @@ impl UnstableAttribute {
...
+                "\n\
                 # Availability\n\
                 \n\
                 **This API is marked as unstable** and is only available when \
                 the `{}` crate feature is enabled. This comes with no stability \
                 guarantees, and could be changed or removed at any time.\
-            ", feature_name);
+            ",
+                feature_name
+            );
```
Comment (loyd) [inline src/unstable.rs:33]: by rustfmt

**Review by sagebind — APPROVED** [review]
Thanks!

**Comment by sagebind** [comment]
This is now published in the 0.1.1 release: <https://crates.io/crates/stability/0.1.1>

**Comment by loyd** [comment]
Thanks a lot!

**Pull request merged**